.github/workflows/checks.yml

```diff
@@ -34,6 +34,10 @@ jobs:
     - name: Output installed packages
       run: |
         pip freeze --all
+    - name: Output dependency tree
+      run: |
+        pip install pipdeptree
+        pipdeptree
```

Comment:
[Notes] Good catch! It's easy to check dependencies on grandchildren. <img width="621" alt="image" src="https://user-images.githubusercontent.com/3255979/233275110-3896f456-4502-4db6-99f7-923d02492d36.png">